cfme/roles.py

```diff
@@ -381,7 +526,7 @@ def _remove_from_all(roles, r_page):
             'Clouds': ['Instances'],
             'Infrastructure': ['Datastores', 'Providers', 'Virtual Machines', 'Hosts', 'Clusters',
                                'PXE', 'Resource Pools'],
-            'Physical Infrastructure': ['Servers', 'Providers']},
+            'Physical Infrastructure': ['Providers', 'Servers', 'Topology']},
```

Comment:
did they change order ?

Reply:
IIRC they didn't change order, but this order is consistent with the UI.